tests/Functional/Filter/ExcludeTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Functional\Filter;
+
+use Paraunit\Filter\Filter;
+use Tests\BaseFunctionalTestCase;
+
+class ExcludeTest extends BaseFunctionalTestCase
+{
+    protected function setup(): void
+    {
+        $this->setOption('configuration', $this->getStubPath() . DIRECTORY_SEPARATOR . 'phpunit_with_2_testsuites.xml');
+        $this->setOption('exclude-testsuite', 'suite2');
+
+        parent::setup();
+    }
```

Comment:
Doing this in the setup is not good: in this way you're not testing the case of multiple, comma-separated values.